src/lib/Scenes/ViewingRoom/Components/ViewingRoomArtworkRail.tsx

```diff
@@ -15,30 +15,33 @@ interface ViewingRoomArtworkRailProps {
 
 export const ViewingRoomArtworkRail: React.FC<ViewingRoomArtworkRailProps> = (props) => {
   const viewingRoom = props.viewingRoom
-  const totalCount = viewingRoom.artworks! /* STRICTNESS_MIGRATION */.totalCount! /* STRICTNESS_MIGRATION */
+  const totalCount = viewingRoom.artworks?.totalCount
   const tracking = useTracking()
   const pluralizedArtworksCount = totalCount === 1 ? "artwork" : "artworks"
```

Comment:
```suggestion
  const pluralizedArtworksCount = totalCount > 1 ? "artworks" : "artwork"
```
because totalCount can be undefined, per your strictness migration above.

Reply:
I think `totalCount === 1 ? "artwork" : "artworks"` is correct. If `totalCount` is `undefined` or `null` (which shouldn't happen) we should render `artworks` and not `artwork`. 

This PR just replaces the `STRICTNESS_MIGRATION` comments with `?.`.